nb_lightning_talks

```diff
@@ -40,6 +58,243 @@ html"""
+- **Append 3 plots**: Three plots are added using `!`(`plot_erp!` and `plot_butterfly!`) with appropiate data and styling.
+- **Display figure**: The figure `f2` is displayed at the end.
+"""
+
+# ╔═╡ 69eb22de-5ee1-4bbb-bb5e-56f6ea9976be
+begin 
+	f2 = Figure(size = (900, 600))
+	with_theme(theme_ggthemr(:fresh)) do
+	    plot_erp!(f2[1, 1], StatsModels.coeftable(uf_deconv), mapping = (; color = :coefname => "Conditions"))
+	    plot_erp!(
+	        f2[1, 2],
+	        Unfold.effects(Dict(:condition => ["car", "face"]), uf_deconv),
+	        mapping = (; color = :condition => "Conditions"),
+	    )
+	    plot_butterfly!(f2[2, 1:2], dat; positions = positions,
+	        topo_attributes = (; label_scatter = (; markersize = 5)),
+	        topo_axis = (; height = Relative(0.5), width = Relative(0.5)))
+	end
+	f2
+end
+
+# ╔═╡ c9330906-bc37-4f17-900a-d56225870c14
+md"""
+More advanced plot
+- **Assign each subplot to a specific variable* for readability**
+- **Set superlables**
+"""
+
+# ╔═╡ 240fcfc2-489d-4ba3-b981-a04f6fa703e1
+begin 
+    f3 = Figure(size = (1200, 800))  # Adjusting figure size for 4 plots
+    
+    # Assigning subplots using ga, gb, gc, and gd
+    ga, gb, gc, gd = f3[1, 1], f3[1, 2], f3[2, 1], f3[2, 2]
+
+    # Applying the theme
+    with_theme(theme_ggthemr(:fresh)) do
+        # Plot ERP on the first subplot (ga)
+        plot_erp!(ga, StatsModels.coeftable(uf_deconv), 
+                  mapping = (; color = :coefname => "Conditions"),
+                  axis = (; xlabel = "Time [ms]"))
+        
+        # Plot Effects on the second subplot (gb)
+        plot_erp!(gb, Unfold.effects(Dict(:condition => ["car", "face"]), uf_deconv), 
+                  mapping = (; color = :condition => "Conditions"),
+                  axis = (; xlabel = "Time [ms]"))
+        
+        # Plot Butterfly on the third subplot (gc)
+        plot_butterfly!(gc, dat; positions = positions,
+                        topo_attributes = (; label_scatter = (; markersize = 5)),
+                        topo_axis = (; height = Relative(0.5), width = Relative(0.5)),
+                        axis = (; xlabel = "Time [ms]"))
+        
+        # Plot Topoplot on the fourth subplot (gd)
+        plot_topoplot!(gd, dat_raw[:, 340, 1]; 
+                       positions = positions,
+                       axis = (; xlabel = "[340 ms]"))
+    end
+    
+    # Adding Labels for Subplots
+    for (label, layout) in zip(["A", "B", "C", "D"], [ga, gb, gc, gd])
+        Label(
+            layout[1, 1, TopLeft()],
+            label,
+            fontsize = 26,
+            font = :bold,
+            padding = (20, 20, 22, 0),
+            halign = :right,
+        )
+    end
+    
+    f3  # Display the final figure
+end
+
+
+# ╔═╡ 12baf0e1-fce3-4645-8cd8-9cdbd60fd9c3
+md"""
+!!! hint "Hint"
+    `(label_scatter = false,)` and `(; label_scatter = false)` are both correct ways to pass named arguments in Julia. 
```

Comment:
this is misleading!

Both of these create a NamedTuple which we require as a named argument. But this is just our usage, so nothing general to julia